frontend/src/components/Header/Logout.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { connect } from 'react-redux';
+import { logout } from '../../flux/actions/authActions'
+import { Link, BrowserRouter } from "react-router-dom";
+
+function Logout(props) {
+  
+  const {logout} = props
+  return (
+    <>
+      <BrowserRouter>
```

Comment:
delete router here aswell

Reply:
thank you, I didn't plan to publish the final version like this of course, wanted to see first that tests are running fine this way or something else is incorrect

Reply:
@tamas-w you can run the tests in this case on your machine :)

Reply:
My explanation was a bit foggy, I always run the tests on my machine first of course. I became happy after every tests went green and I pushed without thinking : )